uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-comments-tab.component.js

```diff
@@ -23,16 +23,14 @@ define([
   function controller ($log, $rootScope, HRSettings, sharedSettings, Contact, Session) {
     $log.debug('Component: leave-request-popup-comments-tab');
 
-    var vm = Object.create(this);
+    var vm = this;
 
+    vm.loading = { component: true };
+    vm.loggedInContactId = null;
```

Comment:
Just last thing, apologies. Could you please make loggedInContactId private variable since it is not needed in the view?